packages/components/src/components/multi-select/multi-select/multi-select-utils.ts

```diff
@@ -163,7 +163,10 @@ export const resetSelectedOptions = (options: MultiSelectOption[]): void =>
     }
   });
 
-export const getNewOptionIndex = (options: MultiSelectOption[], direction: SelectDropdownDirectionInternal): number => {
+export const getNewOptionIndex = (
+  options: MultiSelectOption[],
+  direction: SelectDropdownDirectionInternal
+): number | void => {
```

Comment:
We should use `number | undefined`. In general `void` should only be used if we don't use the return value in any case.

See: https://stackoverflow.com/questions/58885485/why-does-typescript-have-both-void-and-undefined 

Please adjust for all functions